product-scenarios/11-Asynchronous-message-processing/11.1-Decoupling-interacting-parties-across-time-and-space/11.1.1-Decouple-parties-using-JMS/11.1.1.1-HTTP-ActiveMQ-HTTP/11.1.1.1.1-asynchronous-point-to-point-via-queue/src/test/java/org/wso2/carbon/ei/scenario/test/AsyncHttpToHttpViaActiveMqQueueTest.java

```diff
@@ -71,4 +71,33 @@ public void testAsyncHttpToHttpViaQueueAndListeningJMSProxy() throws IOException
         //Assert logs to verify the message is picked by JMS listener proxy from the ActiveMQ queue
         ElasticSearchClient.assertForSingleLogEntry(getElasticSearchHostname(), getDeploymentStackName(),msgID);
     }
+
+    /**
+     * Test Scenario : Receive HTTP request, store in a queue, receive jms message over a jms inbound endpoint and
+     * forward to an http endpoint
+     *
+     * @throws IOException
+     */
+    @Test(description = "11.1.1.1.1.2")
+    public void testAsyncHttpToHttpViaQueueAndReceiveMessageOverInboundEndpoint() throws IOException {
+        UUID uuid = UUID.randomUUID();
+        String msgID = getTestRunUUID() + "_11_1_1_1_1_2_" + uuid;
+        String request =
+                "{" +
```

Comment:
Line wrapping needs to happen before an operator. Refer, https://www.oracle.com/technetwork/java/codeconventions-150003.pdf for code conventions.